openml/extensions/onnx/config.py

```diff
@@ -0,0 +1,49 @@
+import mxnet as mx
+from mxnet import gluon
+
+from openml import OpenMLTask, OpenMLClassificationTask, OpenMLRegressionTask
+
+from typing import Callable
+
+
+# _default_criterion_gen returns a criterion based on the task type - regressions use
+# gluon.loss.L2Loss while classifications use gluon.loss.SoftmaxCrossEntropyLoss
+def _default_criterion_gen(task: OpenMLTask) -> gluon.loss.Loss:
+    if isinstance(task, OpenMLRegressionTask):
+        return gluon.loss.L2Loss()
+    elif isinstance(task, OpenMLClassificationTask):
+        return gluon.loss.SoftmaxCrossEntropyLoss()
+    else:
+        raise ValueError(task)
+
+
+# criterion_gen returns the criterion based on the task type
+criterion_gen = _default_criterion_gen  # type: Callable[[OpenMLTask], gluon.loss.Loss]
+
+
```

Comment:
Many extra blank lines. Can you run the flake8 script and fix these small style issues?